tests/secretize/idempotent.mlir

```diff
@@ -0,0 +1,12 @@
+// RUN: heir-opt -secretize -wrap-generic -secretize %s | FileCheck %s
```

Comment:
Maybe make the test a bit more explicit, as a change/bug in wrap-generic could cause it to regress to double-wrapping even for non-doubled-up `secret<secret<types>>`
```suggestion
// RUN: heir-opt -secretize -wrap-generic -secretize -wrap-generic %s | FileCheck %s
```

Reply:
nice! done. also added a check to ensure that there's only one secret.generic in that case